src/app/VerifyActivities/components/ActionsComponent.js

```diff
@@ -1,11 +1,31 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { ButtonComponent } from '../../common/components';
 
-const ActionsComponent = () => (
+const ActionsComponent = ({ handleVerify, id }) => (
   <div className='actions'>
-    <ButtonComponent className='action--reject'><div className='cross'>&times;</div></ButtonComponent>
-    <ButtonComponent className='action--approve'><div className='checkmark' /></ButtonComponent>
+    <ButtonComponent
+      className='action--reject'
+      onClick={() => handleVerify(id, 'rejected')}
+    >
+      <div className='cross'>&times;</div>
+    </ButtonComponent>
+    <ButtonComponent
+      className='action--approve'
+      onClick={() => handleVerify(id, 'approved')}
```

Comment:
The status here should be `pending`. This is in reference to the activity life cycle

Reply:
@Chris-Maina This is fixed